astroquery/sdss/core.py

```diff
@@ -193,16 +204,27 @@ def query_crossid_async(self, coordinates, *, radius=5. * u.arcsec, timeout=TIME
                                  timeout=timeout, cache=cache)
         return response
 
-    def query_region_async(self, coordinates, *, radius=2. * u.arcsec, timeout=TIMEOUT,
+    def query_region_async(self, coordinates, *, radius=None,
+                           width=None, height=None, timeout=TIMEOUT,
                            fields=None, photoobj_fields=None, specobj_fields=None, obj_names=None,
                            spectro=False, field_help=False, get_query_payload=False,
                            data_release=conf.default_release, cache=True):
-        """
-        Used to query a circular region (a "cone search") around given coordinates.
-
-        This function is equivalent to the object cross-ID (`query_crossid`),
-        with slightly different parameters. It returns all objects within the
-        search radius; this could potentially include duplicate observations
+        """Used to query a region around given coordinates. Either `radius` or
```

Comment:
try 
```suggestion
        """
        Used to query a region around given coordinates. Either `radius` or
```

Reply:
That helped, but now it seems to be having problems with lines like:
```
Note that there is a server-side limit of 3 arcmin on `radius`.
```
which is the standard way to refer to arguments to a function.  Instead it is being interpreted with the `py:obj` role.

Reply:
PS, [The Numpy Style Guide](https://numpydoc.readthedocs.io/en/latest/format.html#parameters) uses single backticks to refer to parameters.

Reply:
We must have (I think unintentionally?) assigned single-backticks to be objects?  I don't think this was an intentional choice.  See the opening note on https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html.

Reply:
Right, I was seeing other evidence of that. Probably not worth trying to solve in this PR, so I'll just use double-backticks for now.

Reply:
If you can figure out where we set that, let me know - this is something that has always confused and frustrated me too.

Reply:
https://github.com/astropy/sphinx-astropy/blob/main/sphinx_astropy/conf/v1.py#L80

Reply:
Documentation is compiling successfully now, & I created astropy/sphinx-astropy#58 to ask about why `default_role` is set.